ssr/src/page/account_help.rs

```diff
@@ -43,30 +43,41 @@ fn MenuFooter() -> impl IntoView {
 }
 
 #[component]
-pub fn AccountHelp() -> impl IntoView {
-    let (is_connected, _) = account_connected_reader();
-    let show_delete_modal = create_rw_signal(false);
-    let show_confirm_delete_modal = create_rw_signal(false);
-    let show_confirm_perma_delete_modal = create_rw_signal(false);
-
+fn ConfirmDeleteModal(#[prop(into)] show: RwSignal<bool>) -> impl IntoView {
     view! {
-        <ShadowOverlay show=show_confirm_delete_modal>
+        <ShadowOverlay show=show>
             <div class="flex flex-col gap-4 p-6 mx-6 w-full lg:w-1/2 max-h-[65%] rounded-xl bg-neutral-900">
                 <h2 class="text-white font-bold text-2xl text-center">Are you sure you want to delete your account?</h2>
                 <a class="bg-[#E2017B] text-white px-0 py-3 rounded-full text-center" href="/logout">Yes</a>
```

Comment:
use pre-existing color classes instead of hardcoding
`color-primary-600`

Reply:
understood